code/modules/organs/organ_external.dm

```diff
@@ -208,8 +211,8 @@
 				burn = max(0, burn - can_inflict)
 		//If there are still hurties to dispense
 		if (burn || brute)
-			if (status & ORGAN_ROBOT && body_part != UPPER_TORSO && body_part != LOWER_TORSO)
-				droplimb(1) //Robot limbs just kinda fail at full damage.
+			if (fail_at_full_damage == 1 && body_part != UPPER_TORSO && body_part != LOWER_TORSO)
```

Comment:
don't think this needs to explicitly be 1; just "fail_at_full_damage", I believe.